presto-product-tests/src/main/java/io/prestosql/tests/hive/TestTablePartitioningSelect.java

```diff
@@ -88,7 +90,7 @@ public Requirement getRequirements(Configuration configuration)
                 mutableTable(SINGLE_INT_COLUMN_PARTITIONED_AVRO, TABLE_NAME, LOADED));
     }
 
-    @Test
+    @Test(groups = STORAGE_FORMATS)
```

Comment:
let's not add it to the group. it's already heavy

Reply:
Sure, removed. It just won't run against hdp3.1 until we add the rest of the tests in.